setup.js

```diff
@@ -0,0 +1,184 @@
+const readline = require('readline');
+const fs = require('fs');
+const { execSync } = require('child_process');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+// Helper function to handle system-specific commands
+function executeSystemCommand(command) {
+  try {
+    execSync(command, { stdio: 'inherit' });
+  } catch (error) {
+    if (error.code === 'ENOENT' || error.code === 'EEXIST') {
+      console.error(error.message);
+      // Try to continue execution even if some non-critical issues occur
+    } else {
+      console.error(error);
+      process.exit(1);
+    }
+  }
+}
+
+rl.question('Name? (This will go on the LICENSE)\n=> ', (name) => {
+  rl.question('Email?\n=> ', (email) => {
+    rl.question('Username? (https://github.com/<username>)\n=> ', (username) => {
+      rl.question('Repository? (https://github.com/$username/<repo>)\n=> ', (repository) => {
+        rl.question('Project name?\n=> ', (proj_name) => {
+          rl.question('Short description?\n=> ', (proj_short_desc) => {
+            rl.question('Long description?\n=> ', (proj_long_desc) => {
+              rl.question('Documentation URL?\n=> ', (docs_url) => {
+                console.log('\n\n===== Log =====');
+                console.log('Name:', name);
+                console.log('Email:', email);
+                console.log('Username:', username);
+                console.log('Repository:', repository);
+                console.log('Project name:', proj_name);
+                console.log('Project short description:', proj_short_desc);
+                console.log('Project long description:', proj_long_desc);
+                console.log('Docs URL:', docs_url);
+                console.log('================');
+
+                rl.question('Confirm? (y/n)\n=> ', (confirm) => {
+                  if (confirm.toLowerCase() === 'y') {
```

Comment:
Change this to a guard clause. It'll be easier to read this way.

```js
if (confirm.toLowerCase()) !== 'y') {
  console.log('Aborted.');
  rl.close();
  process.exit(1);
}

// if "yes" code
```

Reply:
Applied the change.